Stratis.Bitcoin.Tests/RPC/Models/GetInfoModelTest.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Newtonsoft.Json.Linq;
+using Stratis.Bitcoin.RPC.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace Stratis.Bitcoin.Tests.RPC.Models
+{
+    public class GetInfoModelTest : BaseRPCModelTest
```

Comment:
Please pull apart the tests into 4, full/partial serialization to json string. full/partial deserialization from jsonstring to JObject.